### PR TITLE
Switching tree section on landing page

### LINKED ISSUE #1861: Landing Page Update
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Needs to be more prominent

On the Gitcion.co landing page, switch the Tree section and the 1,2,3 section.  Add Icons to the Tree section.

screen shot 2018-07-31 at 1 00 27 am

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Enhancement

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*:

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- [x] Switch the Tree section and the 1,2,3 section. 
- [ ] Add 6 Icons to the Tree section (icons to be provided by @pixelantdesign)
- [x] This page should remain responsive

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

Review with @pixelantdesign

### PR DESCRIPTION
**Pull request by nanspro**
##### Description

Switching tree section with 1,2,3 section on landing page.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines]

##### Affected core subsystem(s)

frontend

##### Testing

Locally

##### Refers

Fixes #1861

### CODE REVIEW

File: app/retail/templates/landing/funder.html [hunk 1]
```diff
@@ -46,20 +46,11 @@
         {% include 'landing/funder/benefits.html' %}
       </div>
 
+
```
Comment (thelostone-mc) [inline app/retail/templates/landing/funder.html]: could we remove the extra line ?

File: app/retail/templates/landing/funder.html [hunk 2]
```diff
@@ -151,6 +142,23 @@ <h5>{% trans "Get Paid." %}</h5>
...
+          {% include 'shared/tree.html' %}
+          <h2 class="mt-4">Grow Open Source</h2>
+        </div>
+      </div>
+      
+      <div class='row'>
+        <div class='col-lg-8 col-lg-offset-2'>
+          <hr>
+        </div>
+      </div>
```
Comment (thelostone-mc) [inline app/retail/templates/landing/funder.html]: Why do we have this extra row ? Could we remove it ?

**Comment by nanspro** [comment]
@thelostone-mc  changes made as suggested

**Comment by thelostone-mc** [comment]
@nanspro I added a few more changes on top of it ^_^
Could you pull that in before adding the icons provided by @PixelantDesign  ?

**Comment by thelostone-mc** [comment]
Hey @nanspro first off thanks for turning this around quick ^_^
I'm gonna go ahead and merge this to avoid conflicts turning up later

Could you shoot up another PR once @PixelantDesign gives you the icons? 

Thanks for contributing and welcome to gitcoin :D

**Pull request merged**